packages/shared/src/authz/policies/index.ts

```diff
@@ -1,8 +1,10 @@
 import { AuthCheckContextLoaders } from '../domain/loaders.js'
+import { canCreateProjectPolicyFactory } from './canCreateProject.js'
 import { canQueryProjectPolicyFactory } from './canQueryProject.js'
 
 export const authPoliciesFactory = (loaders: AuthCheckContextLoaders) => ({
   project: {
+    canCreateProject: canCreateProjectPolicyFactory(loaders),
     canQuery: canQueryProjectPolicyFactory(loaders)
```

Comment:
this was done in a previous PR, but standard CRUD nomenclature would be more intuitive IMO. e.g. Read for retrieving a single item and List for retrieving many